main.tex

```diff
@@ -126,6 +126,7 @@ \section{Introduction} \label{sec:intro}
 The \astropy project aims to provide an open-source and open-development core
 package (\astropypkg) and an ecosystem of \emph{affiliated packages} that
 support astronomical functionality in the \python programming language.
+``Open-development'' describes a process where anybody with an internet connection can suggest changes to the source code and contribute his or her opinion when new features, bug fixes, or other code changes are discussed (see Sect.~\ref{sect:devmod} and \ref{sect:ape} of how this is organized in practice).
```

Comment:
anybody -> anyone
'contribute an opinion' 

remove 'with an internet connection' -- sometimes suggestions are made in person during meetings

Reply:
While that is technically true, meeting as astropy developer at a meeting is A LOT more restrictive than "with an internet connection" (and I doubt there are people who make suggestions at meetings but don't have an internet connection even at the very moment they are talking). I wanted to put the qualifier here to emphasize just how low the bar to comment on something really is.

Reply:
Minor: No hyphen in "open development" (i.e. "open-development" -> "open development). 

More significant: I would say that "open development" is about more than just the code: it's also that the entire dev process is "in the open".  That's close to this but subtly different.  What about changing "new features, bug fixes, or other code changes are discussed" to "new features, bug fixes, other code changes, governance, or any other aspect of the development process is discussed"?

Reply:
I agree that those are important points, but I don't want the wording to be too strong because thee are many (legitimate) cases where the development of a feature is done by a single developer and pushed to a PR once it's done. At that point, anybody can contribute, but not every PR has a WIP label. I like your suggestion though.

Reply:
I don't like the hyphen either. I put it in for consistency with the title of the paper ("open-science") and the abstract ("open-source"). I think those should not have hyphens either, but that's really a case for the language editor.